client/types/balance_transactions.d.ts

```diff
@@ -0,0 +1,5 @@
+export interface BalanceTransaction {
```

Comment:
The convention for JS/TS file names in a React codebase is to have hyphens (-) rather than underscores (_) as separators.

Reply:
Fixed in fbb9018